Bottom: edfb9fdb4e2af9f14d797d41abd25fc782e749bc
Top:    6c30bc51c82b267038ff63a398e6518d7ed6db61
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-05 15:33:29 +0100

Refresh of support-holes-in-input-polygon

---

diff --git a/src/crosshair.c b/src/crosshair.c
index 1d3a081..e8542e0 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -98,7 +98,8 @@ XORPolygon (PolygonTypePtr polygon, LocationType dx, LocationType dy)
     {
       Cardinal next = next_contour_point (polygon, i);
       gui->draw_line (Crosshair.GC,
-                      polygon->Points[i].X + dx, polygon->Points[i].Y + dy,
+                      polygon->Points[i].X + dx,
+                      polygon->Points[i].Y + dy,
                       polygon->Points[next].X + dx,
                       polygon->Points[next].Y + dy);
     }
